Assets/SEE/Game/SceneManipulation/GameNodeMarker.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using SEE.DataModel.DG;
+using SEE.Game.City;
+using SEE.GO;
+using UnityEngine;
+
+namespace SEE.Game.SceneManipulation
+{
+
+    public static class GameNodeMarker
+    {
+        public static GameObject NodeMarker(GameObject parent, Vector3 position, Vector3 worldSpaceScale)
+        {
+            GameObject sphere = GameObject.CreatePrimitive(PrimitiveType.Sphere);
+            Vector3 position = parent.transofrm.position;
+            Vector3 worldSpaceScale = parent.transform.lossyScale;
+
+            // Getter size and position of sphere.
+            float coords = Math.Min(scale.x, scale.y);
+            Vector3 size = new Vector3(coords, coords, coords);
+            Vector3 position = new Vector3(position.x, position.y + scale.y, position.z);

```

Comment:
This will not work when there is an antenna or any other kind of decoration above the game object. As hinted in the issue description, you should use `GetTop()`.